reactfire/firebase-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apiKey": "AIzaSyBg3u1sJlyJwQCE95oSDH_mtLABS-is8ZM",
+  "authDomain": "rxfire-525a3.firebaseapp.com",
+  "databaseURL": "https://rxfire-525a3.firebaseio.com",
+  "projectId": "rxfire-525a3",
+  "storageBucket": "rxfire-525a3.appspot.com",
+  "messagingSenderId": "844180061847",
+  "appId": "1:844180061847:web:400f7142e2d1aaeb"
+}
```

Comment:
Was this added by accident? I don't think we need it here

Reply:
Thought we needed this for a shared config. Will remove since we are using emulators.